build.cmd

```diff
@@ -0,0 +1,7 @@
+:; set -eo pipefail
```

Comment:
These lines seem like a left-over.

Reply:
It is some kind of hack to make `.cmd` work on Linux and MacOS. I do not know how it works... But I think it does 😉

Reply:
*Magic* 😮 - I needed to chmod it to try it.

Reply:
On MacOS it also builds a bunch of files that are not ignored. @RassK could check if the .gitignore needs some updates?